src/Castle.Core/Castle.Core.csproj

```diff
@@ -19,6 +19,7 @@
 		<PublicSign Condition="'$(OS)'=='Unix'">true</PublicSign>
 		<FrameworkPathOverride Condition="'$(OS)'=='Unix'">$(NuGetPackageFolders)microsoft.targetingpack.netframework.v4.6.1\1.0.1\lib\net461\</FrameworkPathOverride>
 		<PackageTags>castle dynamicproxy dynamic proxy dynamicproxy2 dictionaryadapter emailsender</PackageTags>
+		<OutputPath>bin\$(Configuration)\$(TargetFramework)\</OutputPath>
 		<DocumentationFile>$(OutputPath)Castle.Core.xml</DocumentationFile>
```

Comment:
> Will have a fix for this soon.

I thought the fix was going to be removing the `DocumentationFile` since Windsor has XML docs in its packages without this element.

Reply:
Yes, it is actually

```
<GenerateDocumentationFile>true</GenerateDocumentationFile>
```

I must have missed that earlier.